### PR TITLE
[1393][IMP]auditlog_ext_nrq: pre-commit, README

### PR DESCRIPTION
**Pull request by Ryoko04**
Task [1393](https://www.quartile.co/web#id=1393&action=771&model=project.task&view_type=form&menu_id=505)

changed lines:

https://github.com/qrtl/nrq-custom/blob/d5cb7ca44bb9788d31003530c69a1dfd8819f0ea/auditlog_ext_nrq/models/auditlog_log.py#L20

https://github.com/qrtl/nrq-custom/blob/d5cb7ca44bb9788d31003530c69a1dfd8819f0ea/auditlog_ext_nrq/models/auditlog_log.py#L26

### CODE REVIEW

File: auditlog_ext_nrq/models/auditlog_log.py [hunk 1]
```diff
@@ -6,28 +6,33 @@
...
+        string="Log Category",
+        compute="_compute_get_log_category",
         store=True,
     )
 
     @api.multi
-    @api.depends('line_ids')
-    def _get_log_category(self):
+    @api.depends("line_ids")
+    def _compute_get_log_category(self):
```
Comment (yostashiro) [inline auditlog_ext_nrq/models/auditlog_log.py]:
```suggestion
    def _compute_log_category(self):
```

File: auditlog_ext_nrq/models/auditlog_log.py [hunk 2]
```diff
@@ -6,28 +6,33 @@
...
+    log_category = fields.Selection(
+        [
+            ("state_update", "Submitted/Set back to Draft"),
+            ("fields_update", "Fields Update"),
+            ("create", "Record Created"),
+            ("unlink", "Record Unlinked"),
+            ("other", "N/A"),
+        ],
+        string="Log Category",
+        compute="_compute_get_log_category",
```
Comment (yostashiro) [inline auditlog_ext_nrq/models/auditlog_log.py]:
```suggestion
        compute="_compute_log_category",
```

**Review by yostashiro — APPROVED** [review]
LGTM with just some minor adjustments. 👍

File: auditlog_ext_nrq/models/auditlog_log.py (hunk 2 above)
Comment (Ryoko04) [reply auditlog_ext_nrq/models/auditlog_log.py]: @yostashiro I adjusted as you suggested. Thank you for your review.

**Review by yostashiro — APPROVED** [review]

**Pull request merged**